packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -660,6 +707,41 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     };
   }
 
+  private _getPage(itemIndex: number, visibleRect: IRectangle): {
+    itemCount: number;
+    height: number;
+    data?: any;
+  } {
```

Comment:
Aren't we also using that interface in the props? Shouldn't we create an actual `interface` for it?